source/browseMode.py

```diff
@@ -196,6 +196,44 @@ def isAfterSelection(self):
 		caret=self.document.makeTextInfo(textInfos.POSITION_CARET)
 		return self.textInfo.compareEndPoints(caret, "startToStart") > 0
 
+	def _getLabelForProperties(self, labelPropertyGetter):
+		"""
+		Fetches required properties for this L{TextInfoQuickNavItem} and constructs a label to be shown in an elements list.
+		@Param labelPropertyGetter: A callable taking 1 argument.
+			For example, if L{itemType} is landmark, the callable must return the landmark type when the landmark argument is provided to it.
+			Alternative attributes or dictionary keys might be name or value.
+			An expected callable might be the __getattribute__ method on an L{NVDAObject}, or the get method on a L{Dict}.
+			Note that L{Dict.get} doesn't raise an exception, it returns None instead.

```

Comment:
Is there such a callable for objects, instead of __getattribute__? For objects, we're never fetching attributes that an object does not have, so in theory, __getattribute__ is ok.